src/pages/wordTest/WordTestTitle.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import LearnWords from './LearnWords';
+import GoToTest from './GoToTest';
+import '../../styles/WordTestTitle.scss';
+
+const WordTestTitle = ({ test, date }) => {
+    return (
+        <div className="word-test-main-container">
+            <div className="word-test-list-title">
+                <div className="big-title">{test}</div>
+                <div className="small-title">{date}</div>
```

Comment:
```suggestion
                <span className="big-title">{test}</span>
                <span className="small-title">{date}</span>
```
보통 `div`는 여러 태그를 그룹으로 묶을때 사용합니다. 이런 하나의 요소에 디자인 하는 상황에서는 `span` 태그가 적절해보입니다.
* [#참고자료 1](https://www.geeksforgeeks.org/difference-between-div-and-span-tag-in-html/)
* [#참고자료 2](https://blog.hubspot.com/website/span-vs-div#html)